### DISCUSSION TITLE
What happens if an S/MIME-certificate expires?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#43010] [Other Feature Feedback, Questions, & Ideas]
Hi there 😄 --
I am unsure what happens if the certificate used when signing an commit using S/MIME expires?
Is the commit still going to be verified until the certificate gets revoked, even though expiry is reached?

Thanks for clearing this up 🎉

Greetings,
Thomas

**Comment by LuisContrerasGlz** [comment]
If an S/MIME certificate used to sign a commit expires, the commit will no longer be verified by Git. If a commit was signed with an expired certificate, it will be treated as unsigned and any trust established through the certificate will be lost. To verify a commit, it is important to use a valid certificate and revoke it before it expires.
